README.md

```diff
@@ -97,9 +97,14 @@ When running on a unix machine, provide the path to the 'bin' directory.
   export BITESTS_ARTIFACTORY_DOCKER_REPO=build-info-tests-docker
   export BITESTS_ARTIFACTORY_DOCKER_HOST=tcp://127.0.0.1:1234
  ```
+ * For OSX Agents, Run a Socat Container:
```

Comment:
For OSX Agents, Run a Socat Container:
-->
For OSX agents, run a Socat container: